openhands/runtime/action_execution_server.py

```diff
@@ -692,6 +696,18 @@ def close(self):
         help='BrowserGym environment used for browser evaluation',
         default=None,
     )
+    parser.add_argument(
+        '--git-user-name',
+        type=str,
+        help='Git user name for commits',
+        default='openhands',
+    )
+    parser.add_argument(
+        '--git-user-email',
+        type=str,
+        help='Git user email for commits',
+        default='openhands@all-hands.dev',
+    )
```

Comment:
Do we need to modify stuff in docker_runtime so we pass in these arguments from config when launching the action execution server?